### DISCUSSION TITLE
How to conceal spaces before and after bold, italic, strikethrough words?

### DISCUSSION THREAD

**Discussion by liozur** [nvim-neorg/neorg#1044] [Q&A] [answered]
First of all, I want to thank neorg for changing the way I organize my life.

Neorg is wonderful in every way, but as a native Chinese speaker, my neorg files contain many unnecessary spaces. See the image below for an example.

![265249620-3a91c046-700b-4c63-89b0-a7b77efaff9b](https://github.com/nvim-neorg/neorg/assets/88496834/c9e915e6-1eaf-467a-90ce-ac4412f311f2)

Unlike English, Chinese does not require spaces between characters. However, neorg's existing syntax for bold, italic, strikethrough, etc. uses spaces before and after the markup symbols like `*`/`/`/`-`. This results in extra spaces in my neorg files written in Chinese.

I could probably fix this by defining custom conceal rules in nvim, but I'm not very experienced with `lua` or `vim script`. Any suggestions would be greatly appreciated!

**Comment by champignoom** [comment, marked as answer]
use `:*...*:` and `:-...-:`

![image](https://github.com/nvim-neorg/neorg/assets/66909116/33bf43be-83ee-41a8-9d21-954dbcbf1af4)

**Reply by liozur** [reply]
Perfect！
